.travis.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     - stage: commit-OpenAPI
       install: npm i -g swagger2openapi
       script:
-        - swagger2openapi src/main/resources/swagger/ga4gh-tool-discovery.yaml -o src/main/resources/swagger/openapi.yaml
+        - swagger2openapi openapi/ga4gh-tool-discovery.yaml -o openapi/openapi.yaml
```

Comment:
Is this converting 2.0 to 3+? Just confirming.

Reply:
Yes, this is responsible for the conversion that eventually gets checked in as
https://github.com/ga4gh/tool-registry-service-schemas/pull/84/commits/ff486e24f7b257faf8b4ea3bdd90516cc01fa164